src/pages/filter/ui/panels/function-panel/components/compound-request/compound-request.store.ts

```diff
@@ -23,6 +26,9 @@ class CompoundRequestStore {
   private _resetValue: string = ''
   private _activeRequestIndex = this.requestCondition.length - 1
   private _currentMode?: ModeTypes
+  private _approx = datasetStore.isXL
```

Comment:
Looks like somebody switched from C# :) This _privateProp thing doesn't make sense in TS. If you create new props, avoid it.